lib/commands/lib/urls_to_check.json

```diff
@@ -0,0 +1,15 @@
+[
+	"https://api.appcelerator.com/p/v1/release-list",
+	"https://api.appcelerator.net/p/v1/release-list",
+	"https://api.cloud.appcelerator.com",
+	"http://developer.appcelerator.com",
+	"https://preview.appcelerator.com",
+	"http://preview.appcelerator.com",
+	"https://studio.appcelerator.com",
+	"http://studio.appcelerator.com",
+	"https://www.appcelerator.com",
+	"http://www.appcelerator.com",
+	"https://www.google.com",
+	"https://github.com",
+	"https://registry.npmjs.org"
+]
```

Comment:
Thanks for incorporating the changes. We don't need to add "http" protocol for dashboard.appcelerator.com